recipes/libcusolver/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+# cuSOLVER ships a copy of METIS, but this is not OK from the conda-forge perspective
+rm lib/libmetis_static.a
```

Comment:
I added this because on conda-forge this is supposed to be generated from metis-feedstock, not from libcusolver-feedstock which is totally unexpected. While cuSOLVER needs METIS for its reason, this is surprising from conda-forge perspective. We are lucky here even if I don't remove it, though, because metis-feedstock is currently not generating static libs. But it's not guaranteed.

However, there's `libcusolver_metis_static.a` (I didn't remove) and I worry that if this one is removed, `libcusolver_metis_static.a` is rendered useless. Perhaps we should also remove it for the time being?

Reply:
@eriknw, do you have any thoughts on how METIS is handled? No worries if not (just thought this might be of interest)

Reply:
cc @conda-forge/metis (in case any of you have thoughts here)

Reply:
Since we place everything under targets now, this discussion is moot.